doc-files/release-notes-23.md

```diff
@@ -0,0 +1,145 @@
+# Release Notes for JavaFX 23
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+These release notes cover the standalone JavaFX 23 release. JavaFX 23 requires JDK 21 or later.
+
+## Important Changes
+
+### JavaFX 23 Requires JDK 21 or Later
```

Comment:
why are `Requires` and `Later` capitalised?

I went back and it was also the case with [JFX 23 release notes](https://github.com/openjdk/jfx/pull/1563/files#diff-2d16210695314e7aef40df186b6428372f477736598ee49496f5340159d72fcbR11). Just curious is there a defined casing style that we follow for release notes?

Reply:
This is the PR for the JavaFX 23 release notes. Did you mean to ask your question in PR #1712 , which is under review for the JavaFX 24 release notes? In any case, I'll answer your question here.

> why are `Requires` and `Later` capitalised?
> 
> I went back and it was also the case with [JFX 23 release notes](https://github.com/openjdk/jfx/pull/1563/files#diff-2d16210695314e7aef40df186b6428372f477736598ee49496f5340159d72fcbR11). Just curious is there a defined casing style that we follow for release notes?

See [this section of the OpenJDK Developers Guide](https://openjdk.org/guide/#general-conventions-for-release-notes).

```
The Summary should be in title case instead of sentence case.

    Example: Decode Error with Tomcat Version 7.x
```

Reply:
> This is the PR for the JavaFX 23 release notes

My bad.

> See [this section of the OpenJDK Developers Guide](https://openjdk.org/guide/#general-conventions-for-release-notes).

Thank you for the link.